README-fr.adoc

```diff
@@ -0,0 +1,310 @@
+= Asciidoctor
+Dan Allen <https://github.com/mojavelinux[@mojavelinux]>; Sarah White <https://github.com/graphitefriction[@graphitefriction]>; Ryan Waldron <https://github.com/erebor[@erebor]>
+
+include::_settings-README.adoc[]
+
+{uri-project}/[Asciidoctor] est un processeur de texte _rapide_ et une chaîne de publication pour convertir du contenu {uri-what-is-asciidoc}[AsciiDoc] en HTML5, DocBook 5 (ou 4.5) et d'autres formats.
+Asciidoctor est écrit en Ruby, packagé sous forme de RubyGem et publié sur {uri-rubygem}[RubyGems.org].
+La gem est aussi incluse dans plusieurs distributions Linux, incluant Fedora, Debian et Ubuntu.
+Asciidoctor est open source, {uri-repo}[hébergé sur GitHub] et releasé sous {uri-license}[la licence MIT].
+
+.Documentation clée
+[.compact]
+* {uri-docs}/what-is-asciidoc[Qu'est ce qu'Asciidoc ?]
+* {uri-docs}/asciidoc-writers-guide[Guide pour Ecrivain AsciiDoc]
+* {uri-docs}/asciidoc-syntax-quick-reference[Syntaxe de Référence Asciidoc]
+* {uri-docs}/user-manual[Manuel Utilisateur Asciidoctor]
+
+.Là où Ruby va, Asciidoctor le suit
+****
+Vous pouvez exécuter Asciidoctor dans le JVM en utilisant JRuby.
+Pour invoquer l'API Asciidoctor directement de Java ou d'autres langages de JVM, utilisez {uri-asciidoctorj}[AsciidoctorJ].
+Il y a des plugins disponibles, basés sur AsciidoctorJ, qui intègrent le processeur Asciidoctor dans des builds Apache Maven, Gradle ou Javadoc.
+
+Asciidoctor s'exécute également au sein de JavaScript.
+Nous utilisons http://opalrb.org[Opal] pour transpiler le code source Ruby en JavaScript afin de produire {uri-asciidoctorjs}[Asciidoctor.js], une version pleinement fonctionnelle d'Asciidoctor qui s'intègre dans tout environnement JavaScript, comme un navigateur web ou Node.js.
```

Comment:
`transcrire` instead of `transpiler` even if I'm not 100% this word should be translated